go.mod

```diff
@@ -1,17 +1,19 @@
 module knative.dev/func
 
-go 1.22
+go 1.22.0
```

Comment:
@dsimansk  Now `./hack/update-deps.sh` did this. I have no idea why.

Reply:
It'll some default behavior change / more strict rules from Go 1.22, I guess.